man/OptSpace.Rd

```diff
@@ -0,0 +1,77 @@
+\encoding{UTF-8}
+\name{OptSpace}
+\alias{OptSpace}
+\title{OptSpace: algorithm for matrix reconstruction from a partially revealed set}
+\description{
+This function was adapted from the original source code in the
+ROptSpace R package (version 0.2.3; MIT License) by Raghunandan
+H. Keshavan, Andrea Montanari, Sewoong Oh (2010). See
+ROptSpace::OptSpace for more information.  Let's assume an ideal
```

Comment:
`ROptSpace::OptSpace` -> `\code{ROptSpace::OptSpace}`, assuming we don't want to link to the function in the package (if we do then the package would have to be `Suggests`, which I think we wanted to avoid?)